### PR TITLE
[SIDE-2274] Replace p with div to address facebook/react#5479

### PR DESCRIPTION
**Pull request by richseviora**
Disappearing text issue caused by nested `p` tag reconciliation issue. Also making this branch trigger doc updates to test.

### CODE REVIEW

**Review by cshields-oliveai — APPROVED** [review]

File: .github/workflows/prettier.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Prettier
```
Comment (flowermm) [inline .github/workflows/prettier.yml:1]: Happy to see Prettier added!

File: ldk/javascript/src/webpack/config.ts [hunk 2]
```diff
@@ -2,9 +2,9 @@ import * as webpack from 'webpack';
 import * as path from 'path';
 import Terser from 'terser-webpack-plugin';
 import { generateBanner } from './generate-banner';
-import { LdkSettings } from './ldk-settings';
+import { LdkSettings } from './ldk-settings'; // Need to dynamically refer to Loop's package.json
 
-/* eslint-disable-next-line */ // Need to dynamically refer to Loop's package.json
```
Comment (flowermm) [inline ldk/javascript/src/webpack/config.ts:7]: I realize that the bulk of this PR is prettier doing its thing. Is it worth keeping why we need to disable the linter? Maybe change the eslint disable to reference the exact rule we are breaking

**Review by flowermm — APPROVED** [review]
Glad to see the Gatsby/React issue was a simple fix! Also excited for prettier to help keep things looking uniform 👍

File: .github/workflows/prettier.yml (hunk 1 above)
Comment (richseviora) [reply .github/workflows/prettier.yml:1]: In addition to being a Typescript fan boy I am _also_ a Prettier fanboy 😀

File: ldk/javascript/src/webpack/config.ts (hunk 2 above)
Comment (richseviora) [reply ldk/javascript/src/webpack/config.ts:7]: Ooo yes, great point, I try to document the specific rule and why I'm suppressing it, and I should do that here too.

Comment (richseviora) [reply ldk/javascript/src/webpack/config.ts:7]: Is done!

**Pull request merged**